main_test.go

```diff
@@ -849,3 +899,235 @@ func Example_ShowOrdersProductIDFlag_CustomerIDFlag() {
+	_, err = db.Query("SELECT * FROM Products")
+	assert.NilError(t, err)
+	_, err = db.Query("SELECT sku FROM Products")
+	assert.NilError(t, err)
+}
+
+func runMigrationsHelper(t *testing.T) {
+	ctx, cancel := context.WithTimeout(context.Background(), time.Second*30)
+	defer cancel()
+	app := &cli.App{
+		Commands: []*cli.Command{
+			runMigrations(ctx),
+		},
+	}
+	app.Run([]string{"store", "run-migrations"})
+}
+
+func TestConnectingToDefault(t *testing.T) {
+	var db *sql.DB
+	app := &cli.App{
+		Flags: []cli.Flag{
+			&cli.StringFlag{
+				Name:  "org",
+				Usage: "org to connect to",
+				Value: "default",
+			},
+		},
+		Before: func(cCtx *cli.Context) error {
+			org := cCtx.String("org")
+			log.Println("connecting to org:", org)
+
+			orgDB, err := connectDB(org)
+			if err != nil {
+				return err
+			}
+
+			db = orgDB
+			return nil
+		},
+	}
+	assert.NilError(t, app.Run([]string{"store"}))
+	_ = db
+}
+
+func TestConnectingToMicrosoft(t *testing.T) {
+	var db *sql.DB
+	app := &cli.App{
+		Flags: []cli.Flag{
+			&cli.StringFlag{
+				Name:  "org",
+				Usage: "org to connect to",
+				Value: "default",
+			},
+		},
+		Before: func(cCtx *cli.Context) error {
+			org := cCtx.String("org")
+			log.Println("connecting to org:", org)
+
+			orgDB, err := connectDB(org)
+			if err != nil {
+				return err
+			}
+
+			db = orgDB
+			return nil
+		},
+	}
+	assert.NilError(t, app.Run([]string{"store", "--org=microsoft"}))
+	_ = db
+}
+
+func TestConnectingToGoogle(t *testing.T) {
+	var db *sql.DB
+	app := &cli.App{
+		Flags: []cli.Flag{
+			&cli.StringFlag{
+				Name:  "org",
+				Usage: "org to connect to",
+				Value: "default",
+			},
+		},
+		Before: func(cCtx *cli.Context) error {
+			org := cCtx.String("org")
+			log.Println("connecting to org:", org)
+
+			orgDB, err := connectDB(org)
+			if err != nil {
+				return err
+			}
+
+			db = orgDB
+			return nil
+		},
+	}
+	assert.NilError(t, app.Run([]string{"store", "--org=google"}))
+	_ = db
+}
+
+func TestConnectingToNonExistingDB(t *testing.T) {
+	var db *sql.DB
+	app := &cli.App{
+		Flags: []cli.Flag{
+			&cli.StringFlag{
+				Name:  "org",
+				Usage: "org to connect to",
+				Value: "default",
+			},
+		},
+		Before: func(cCtx *cli.Context) error {
+			org := cCtx.String("org")
+			log.Println("connecting to org:", org)
+
+			orgDB, err := connectDB(org)
+			if err != nil {
+				return err
+			}
+
+			db = orgDB
+			return nil
+		},
+	}
+	assert.ErrorContains(t, app.Run([]string{"store", "--org=abc"}), "Error 1049 (42000): Unknown database 'store_abc'")
+	_ = db
+}
+
+func Test_SuiteOfTests_Microsoft(t *testing.T) {
+	runMigrationsHelper(t)
+	dbName = "microsoft"
+	Test_RunSuiteOfTests(t)
+}
+
+func Test_SuiteOfTests_Google(t *testing.T) {
+	runMigrationsHelper(t)
+	dbName = "google"
+	Test_RunSuiteOfTests(t)
+}
+
+func Test_SuiteOfTests_Default(t *testing.T) {
+	runMigrationsHelper(t)
+	dbName = "default"
+	Test_RunSuiteOfTests(t)
+}
+
+func Test_RunSuiteOfTests(t *testing.T) {
```

Comment:
This is actually redundant - we can just run `go test` from the root directory and all of these tests will run. Also, in vscode, there's options to run all tests in file / run all tests in package - just check at the top of the file.